docs/content/index.md

```diff
@@ -2,60 +2,153 @@
+
+### Real-time Processing Nodes
+
+Real-time processing in Druid can currently be done using standalone realtime nodes or using the indexing service. The real-time logic is common between these two services.
+Real-time processing involves ingesting data, indexing the data (creating segments), and handing segments off to historical nodes. Data is queryable as soon as it is
+ ingested by the realtime processing logic. The hand-off process is also lossless; data remains queryable throughout the entire process.
+
+### External Dependencies
+
+Druid has a couple of external dependencies for cluster operations.
+
+* **Distributed Coordination** Druid relies on a distributed coordination service for intra-cluster communication. The only supported distributed coordination service right now is Zookeeper.
 
-Druid is partially inspired by search infrastructure and creates mostly immutable views of data and stores the data in data structures highly optimized for aggregations and filters. A Druid cluster is composed of various types of nodes, each designed to do a small set of things very well.
+* **Metadata Storage** Druid relies on a metadata storage to store metadata about segments and configuration. Services that create segments write new entries to the metadata store
+  and the coordinator nodes monitor the metadata store to know when new data needs to be loaded or old data needs to be dropped. The metadata store is not
+  involved in the query path. MySQL and PostgreSQL are popular metadata stores.
 
-Druid vs…
-----------
+* **Deep Storage** Deep storage acts as a permanent backup of segments. Services that create segments upload segments to deep storage and historical nodes download
+segments from deep storage. Deep storage is not involved in the query path. S3 and HDFS are popular deep storages.
 
-* [Druid-vs-Impala-or-Shark](Druid-vs-Impala-or-Shark.html)
-* [Druid-vs-Redshift](Druid-vs-Redshift.html)
-* [Druid-vs-Vertica](Druid-vs-Vertica.html)
-* [Druid-vs-Cassandra](Druid-vs-Cassandra.html)
-* [Druid-vs-Hadoop](Druid-vs-Hadoop.html)
-* [Druid-vs-Spark](Druid-vs-Spark.html)
-* [Druid-vs-Elasticsearch](Druid-vs-Elasticsearch.html)
+### High Availability Characteristics
 
-About This Page
-----------
-The data infrastructure world is vast, confusing and constantly in flux. This page is meant to help potential evaluators decide whether Druid is a good fit for the problem one needs to solve. If anything about it is incorrect please provide that feedback on the mailing list or via some other means so we can fix it.
+Druid is designed to have no single point of failure. Different node types are able to fail without impacting the services of the other node types. To run a highly available Druid cluster, you should have at least 2 nodes of every node type running.
 
+### Comprehensive Architecture
 
+For a comprehensive look at Druid architecture, please read our [white paper](http://static.druid.io/docs/druid.pdf).
```

Comment:
When in the doc flow do we expect people to read the white paper? Is it here? If so, we should keep this here. But if not we should move this link somewhere else.